apps/electron/tests/basic.spec.ts

```diff
@@ -19,5 +37,30 @@ test('new page', async () => {
     () => globalThis.currentWorkspace.flavour
   );
   expect(flavour).toBe('local');
-  await electronApp.close();
+});
+
+test('app theme', async () => {
+  await page.waitForSelector('v-line');
+  const root = page.locator('html');
+  {
+    const themeMode = await root.evaluate(element =>
+      element.getAttribute('data-theme')
+    );
+    expect(themeMode).toBe('light');
+  }
+  const prev = await page.screenshot({
+    path: resolve(testResultDir, 'affine-light-theme-electron.png'),
+  });
+  await page.getByTestId('change-theme-dark').click();
+  await page.waitForTimeout(50);
+  {
+    const themeMode = await root.evaluate(element =>
+      element.getAttribute('data-theme')
+    );
+    expect(themeMode).toBe('dark');
+  }
+  const after = await page.screenshot({
+    path: resolve(testResultDir, 'affine-dark-theme-electron.png'),
+  });
+  expect(prev).not.toEqual(after);
```

Comment:
The return result of page.screenshot is a Buffer and comparing them will always be false. 
I guess we may want to have a better image diff solution to solve this issue, but I guess we do not have to be part of Electron test suite. This can be done in the storybook or web instead.